portality/templates/publisher/preservation.html

```diff
@@ -0,0 +1,117 @@
+{% extends "publisher/publisher_base.html" %}
+
+{% block page_title %}Upload article XML{% endblock %}
+
+{% block publisher_content %}
+
+<div class="row">
+  <header class="col-md-4">
+    <h2>Upload preservation file</h2>
+
+    <div class="alert">
+      <ul>
+        <li>Know all about preservation <a href="/preservation">here</a></li>
+        <li>Check help documentation for creating preservation package <a href="{{ url_for('publisher.help') }}#preservation">here</a></li>
+        <li> <a href="/static/doaj/preservation_example.zip">Download </a>example zip file</li>
+      </ul>
```

Comment:
Can we make the whole text inside <li>s the links, please? Something like:
`<li>a href="/preservation"> Click here to learn all about preservation</a></li>`
From accessibility point of view it is advisable to make the link text meaningful.